src/common/xo/index.js

```diff
@@ -18,7 +18,6 @@ import { createBackoff } from 'jsonrpc-websocket-client'
 import { lastly, reflect, tap } from 'promise-toolbox'
 import { forbiddenOperation, noHostsAvailable } from 'xo-common/api-errors'
 import { resolve } from 'url'
-
```

Comment:
[Keep it](https://gitlab.com/vates/company/wikis/Development-Guidelines#imports).